src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * This is a main controller class that handles inputs and changes made by the user
+ * in the UI, and determines which use case method to call
+ */
+package controller;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import ui.ViewLoginPage;
+import usecases.UseCaseFalseLogin;
+import usecases.UseCaseLogin;
+import usecases.UseCaseRegister;
+
+public class Controller {
+    /**
+     * The Controller for the loginButton takes in inputs from the text fields
+     * of the given UI page, and determine which use case to select.
+     */
+    public static void loginButton(ViewLoginPage page) {
+        String username = page.usernameField.getText();
+        String password = String.valueOf(page.passwordField.getPassword());
+
+        if(existsInDatabase(username)){
+            if(passwordCorrectness(password)){
+                UseCaseLogin.login(page);
+            }
+            else{
+                UseCaseFalseLogin.falseLogin(page);
+            }
+        }
+        else{
+            UseCaseRegister.pleaseSignup(page);
+        }
+    }
+
+    /**
+     * The Controller for the signupButton takes in inputs from the text fields
+     * of the given UI page, and determine which use case to select.
+     */
+    public static void signupButton(ViewLoginPage page){
+        String username = page.usernameField.getText();
+        String password = String.valueOf(page.passwordField.getPassword());
+        String email = page.emailField.getText();
+
+        if(existsInDatabase(username)){
+            UseCaseRegister.usernameExists(page);
+        }
+        else{
+            UseCaseRegister.newUser(username, password, email);
+            UseCaseRegister.Registered(page);
+        }
+    }
+
+    /**
+     * This method determine whether the given input from the UI to the username
+     * text field exists in the database
+     *
```

Comment:
Again adding an @return tag to the javadoc for methods that return something  might be a good idea